src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -350,7 +350,7 @@ export class StudySummaryTab extends React.Component<IStudySummaryTabProps, {}>
                                         onChange={this.props.store.toggleWithCNADataFilter}
                                         disabled={this.props.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples === undefined}
                                     >
-                                        {this.props.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples === undefined ? '0' : this.props.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples.toLocaleString()} w/ CNA data
+                                        {this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined ? '0 sample' : this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples.toLocaleString() + (this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples <= 1 ? " sample" : " samples")} with CNA data
```

Comment:
both here and above, the english idiom is "0 samples, 1 sample, 2(+) samples". doesnt make much sense but thats how it is :D

Reply:
I actually just knew. In this case, there are quite few places in the study view need to be updated.